src/Conways/Game/Of/Life/GridView.java

```diff
@@ -11,29 +11,49 @@
 
 public class GridView extends JComponent {
 
-    //private final Grid grid;
-    public static final int CELL_SIZE = 10;
+    public static final int CELL_SIZE = 15;
+    public static final int BORDERED_CELL_SIZE = 17;
+    public static final int LINE_SIZE = 1;
+    public static final int HEIGHT = 30;
+    public static final int WIDTH = 50;
+    public static final Color LIVE_COLOR = Color.ORANGE;
+    public static final Color DEAD_COLOR = Color.GRAY;
+    public static final Color LINE_COLOR = Color.BLACK;
+    Grid grid;
 
     public GridView(Grid grid) {
+        this.grid = grid;
     }
 
     @Override
     protected void paintComponent(Graphics g) {
         super.paintComponent(g);
         paintGrid(g);
-        paintLive(g);
-        paintDead(g);
-
-    }
-
-    //Changes color of cells that don't have sufficient neighbors
-    private void paintDead(Graphics g) {
+        //paintLifeStatus(g);
     }
 
-    private void paintLive(Graphics g) {
+    private void paintLifeStatus(Graphics g) {
+        for (int i = 0; i < WIDTH; i++) {
+            for (int j = 0; j < HEIGHT; j++) {
+                //not sure why ternary wasn't working
+                if (grid.getCells(i, j)) {
+                    g.setColor(LIVE_COLOR);
+                } else {
+                    g.setColor(DEAD_COLOR);
+                }
+                g.fillRect(i * BORDERED_CELL_SIZE, j * BORDERED_CELL_SIZE, CELL_SIZE, CELL_SIZE);
+            }
+        }
     }
 
     private void paintGrid(Graphics g) {
```

Comment:
What is this method for? Isn't paintLifeStatus painting the grid?

Reply:
Stepping stone while there's no grid logic. Probably won't make it to production
\